pkg/skaffold/color/formatter_test.go

```diff
@@ -67,6 +67,22 @@ func TestFprintf(t *testing.T) {
 	compareText(t, "\033[32mIt's been 1 week\033[0m", b.String(), 25, n, err)
 }
 
+type nopCloser struct{ io.Writer }
+
+func (n *nopCloser) Close() error { return nil }
+
+func TestFprintOnColoredWriter(t *testing.T) {
+	var b bytes.Buffer
+
+	coloredWriter := ColoredWriteCloser{
+		WriteCloser: &nopCloser{Writer: &b},
+	}
+
+	n, err := Green.Fprint(coloredWriter, "It's not easy being")
```

Comment:
being what? or just existing in general? :)

Reply:
I think you have to ask @bobcatfish who wrote the original test!

Reply:
bahahaha

Reply:
![image](https://user-images.githubusercontent.com/432502/58660150-9a50a000-82d9-11e9-9458-4d10c1a51b90.png)